[IMP] Events: Update revenues report 18.0 (#19169)
odoo/documentation: content/applications/marketing/events/event_reporting/revenues_report.rst

Revenues`.

.. image:: revenues_report/default-view.png
   :alt: The default view of the Revenues reporting page in the Odoo Events application.

By default, the :guilabel:`Revenues` reporting page appears as a graph (a :icon:`fa-line-chart`
:guilabel:`(Line Chart)` with :icon:`fa-database` :guilabel:`(Stacked)` data). The default filters,
:guilabel:`Non-free tickets` and :guilabel:`Event Start Date: (current year)`, are applied in the
search bar.

.. tip::
   To learn more about the various graph views (and graph view options), refer to the :ref:`Graph
   views <reporting/using-graph>` documentation.

The *Revenues* reporting page can also be viewed as a :ref:`pivot table <reporting/views/pivot>`, by
clicking the :icon:`oi-view-pivot` :guilabel:`(Pivot)` icon in the upper-right corner.

Measures
--------

Choosing specific :ref:`measures <reporting/choosing-measures>` allows users to customize reporting
pages.

Clicking the :guilabel:`Measures` drop-down button reveals the following options:

- :guilabel:`Revenues`: Shows the revenues generated from events.
- :guilabel:`Untaxed Revenues`: Shows the untaxed revenues generated from events.

> - :guilabel:`Count`: Shows the total amount of registrants who attended events.

Reviewer: ```suggestion
- :guilabel:`Count`: Shows the total number of registrants who attended events.
```

Filter options
~~~~~~~~~~~~~~

The :guilabel:`Filters` column in the search bar's drop-down menu allows users to restrict the data
to specific criteria.

Multiple options in the :guilabel:`Filters` column can be selected at once. Options can be applied
to any view.

The :guilabel:`Filters` column has the following options:

> - :guilabel:`Non-free tickets`: Filter by event tickets/registrations that are not free.

Reviewer 2: are these specifically tied to a purchase through the sales app? if so, I would specify that.

- :guilabel:`Registration Status`: Group data based on the status of registrations.
- :guilabel:`Sale Order Status`: Group data based on the status of event-related sales orders.
- :guilabel:`Customer`: Group data based on customer records.
- :guilabel:`Add Custom Group`: Create a :ref:`custom group <search/group>` to organize data by
  additional fields. Click the :icon:`fa-caret-down` :guilabel:`(down arrow)` icon to reveal a
  drop-down of grouping options. Multiple selections can be made.

Sample report: event ticket analysis (graph)
============================================

The following is an example of how various filters and grouping options can be used to compare the
distributions between paid tickets and free tickets for published events, organized by ticket type

> and event.

Reviewer: ```suggestion
distributions of paid tickets and free tickets for published events, organized by ticket type and
event.
```
"Distributions between" reads awkwardly to me.

> To create such this report, navigate to :menuselection:`Events app --> Reporting --> Revenues`. Stay

Reviewer 2: ```suggestion
To create this report, navigate to :menuselection:`Events app --> Reporting --> Revenues`. Stay
```

in the default graph view but remove the default filters from the search bar.

Then, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` next to the search bar. From here,
select :guilabel:`Free` and :guilabel:`Sold` from the :guilabel:`Filters` column.

To view data related **only** to already published events, select the :guilabel:`Published Events`
option in the :guilabel:`Filters` column.

Next, in the :guilabel:`Group By` column, select the :guilabel:`Event` and :guilabel:`Ticket`
options, **in that order**. Doing so first groups the data by event, then by ticket type.

.. important::
   The order in which the options are selected in the :guilabel:`Group By` column directly affects
   how the data is grouped and displayed on the report.

.. image:: revenues_report/event-ticket-analysis.png
   :alt: The event ticket analysis sample report with unique filters and groupings in place.

Sample report: event type analysis (pivot table)
================================================

The following is an example of how to create a report analyzing revenue by different event types in

> order to determine which event types are the most profitable.

Reviewer 2: ```suggestion
The following is an example of how to create a report analyzing revenue by event type to determine which event types are most profitable.
```

First, navigate to :menuselection:`Events app --> Reporting --> Revenues` and switch to the pivot
table view by clicking the :icon:`oi-view-pivot` :guilabel:`(Pivot)` icon in the upper-right corner.

Keep the default filters (:guilabel:`Non-free tickets` and :guilabel:`Event Start Date: (year)`) in
the search bar.

> Next, open the :guilabel:`Measures` drop-down menu, and deselect the option for :guilabel:`Count` as

Reviewer: ```suggestion
Next, open the :guilabel:`Measures` drop-down menu and deselect the option for :guilabel:`Count` as
```
